doc/nrf/ug_thingy53.rst

```diff
@@ -100,7 +100,9 @@ Building and programming using Visual Studio Code
 
 Complete the following steps after installing the `nRF Connect for Visual Studio Code`_ extension in Microsoft Visual Studio Code:
 
-.. include:: /includes/vsc_build_and_run_nrf53.txt
+.. |sample_path_vsc| replace:: :file:`nrf/samples/bluetooth/peripheral_lbs`
```

Comment:
We call the /includes/vsc_build_and_run_nrf53.txt in ug_nrf5340 - http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf5340.html#building-and-programming-a-sample.
Could make the changes there also.

Reply:
I've completely removed the nrf53.txt and used it as a basis for the txt include for the 4 files edited in this PR.